include/votca/xtp/qmmolecule.h

```diff
@@ -22,14 +22,25 @@
 
 #include <votca/xtp/atomcontainer.h>
 #include <votca/xtp/qmatom.h>
+#include <votca/tools/structureparameters.h>
+#include <votca/tools/unitconverter.h>
 
 namespace votca {
 namespace xtp {
 
 class QMMolecule : public AtomContainer<QMAtom> {
  public:
+  typedef QMAtom bead_t;
+
+  static const tools::DistanceUnit distance_unit;                                  
```

Comment:
what is this?

Reply:
This is used by the file readers and writers to resolve the types stores in each of the containers.